docs/book/src/user/quick-start.md

```diff
@@ -107,14 +107,10 @@ Download the latest release; on linux, type:
 ```bash
 curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"1.2.x"}} -o clusterctl
 ```
-Make the clusterctl binary executable.
-```bash
-chmod +x ./clusterctl
+Install clusterctl
 ```
```

Comment:
Hi @hackeramitkumar! 

Thank you for the pr. It looks like formatting isn't correct. Here is an example of a correctly formatted command that you can look at: 
https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/clusterctl/developers.md?plain=1#L15-L17

And for more information you can read: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Thanks, @oscr for your suggestions. I have now made changes. Please review that. And if any changes are required then let me know.